class.jetpack.php

```diff
@@ -3931,9 +3931,10 @@ function admin_page_load() {
 		}
 
 		if ( isset( $_GET['connect_url_redirect'] ) ) {
+			// @todo: Add validation against a known whitelist
```

Comment:
What do you think about sanitizing that key with `sanitize_key()` for now? We could do that in `build_connect_url` as the parameter is added to the final URL.

Reply:
@jeherve I like the idea, but also FWIW I wouldn't block the PR for this, as it wasn't introduced here.

Reply:
true. ✅

Reply:
Pushed #12380 to take care of that.